sample/src/main/java/com/stepstone/stepper/sample/DifferentNextButtonStepperActivity.java

```diff
@@ -60,9 +60,9 @@ public void onBackPressed() {
         }
     }
 
-    private static class MyStepperAdapter extends AbstractStepAdapter {
+    private static class MyStepperAdapterFragment extends AbstractFragmentStepAdapter {
```

Comment:
Probably this should be named MyStepperFragmentAdapter instead of MyStepperAdapterFragment?